generate_openj9_scc.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
+
+# Format taken from OpenLiberty SCC generation in docker images
+
+unset OPENJ9_JAVA_OPTIONS
+
+SCC_SIZE="50m" # Default size for SCC
+SCC_GEN_RUNS_COUNT=3 # Runs for generating SCC
+
+run_eclipse_and_stop() {
+
+    "$HOME"/eclipse-home/eclipse/eclipse &
+    ECLIPSE_PID=$!
+    
+    sleep 1m
+
+    kill -9 $ECLIPSE_PID
+    
+    sleep 10s
+
+}
+
+download_eclipse() {
+
+    # Creating a temporary directory for tomcat download
+    mkdir -p "$HOME"/eclipse-home/
+
+    # Downloading eclipse
+    wget --directory-prefix=/tmp  http://www.mirrorservice.org/sites/download.eclipse.org/eclipseMirror/technology/epp/downloads/release/2020-06/M3/eclipse-java-2020-06-M3-linux-gtk-x86_64.tar.gz
+
+    # Extracting eclipse
+    tar -xvzf /tmp/eclipse-java-2020-06-M3-linux-gtk-x86_64.tar.gz -C "$HOME"/eclipse-home
+
+    # Removing the tar file
+    rm -rf /tmp/eclipse-java-2020-06-M3-linux-gtk-x86_64.tar.gz
+
+}
+
+install_packages() {
```

Comment:
Can the function name be more descriptive. It needs to be clear which packages (eg related to eclipse) and which OSes, since you need to invoke the package manager.

Reply:
Made this function generic now. based on the platform I'm installing the packages and added comments for it. please review it and lemme know if any changes are required